scripts/validate-yaml.py

```diff
@@ -18,6 +18,8 @@ def _check_partition(self, action):
         assert 'ptable' in self.actions[action['device']]
         if action['flag'] in ('boot', 'bios_grub', 'prep'):
             assert self.actions[action['device']]['type'] == 'disk'
+        if action.get('preserve', True):
+            assert 'number' in action
```

Comment:
My preference would be to always have number present for partition.  There's no reason not to set this value.

Reply:
OK.